client/src/utils/getDates.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable arrow-body-style */
+const daysController = (days, delta) => {
+  let alteredDays;
+
+  if (!delta) {
+    const today = new Date();
+    const tomorrow = new Date(new Date().getTime() + 24 * 60 * 60 * 1000);
```

Comment:
I'm curious if there's a better way to do this?

Reply:
I wondered about this, but I left it for the sake of clarity. 
If you look at the day controller, you'll see I used a different way to change the date but that stays in seconds format. And that's the state I keep for modifying. The abbreviated way is needed for rendering.